plugins/EpadDB.js

```diff
@@ -5698,8 +5745,10 @@ async function epaddb(fastify, options, done) {
               reply
                 .code(200)
                 .send(`Aim deleted from system as it didn't exist in any other project`);
-            } else
+            } else {
```

Comment:
delete project and deleteRelationAndOrphanedCouchDocInternal should make sure to update the project_ids in couchdb when deleted